api/rooms/tests/test_map.py

```diff
@@ -1 +1,189 @@
-from django.test import TestCase
\ No newline at end of file
+from django.test import TestCase, Client
+
+from api.rooms.models import Location, RoomType, TradeType, Room, OptionType, RoomImage
+
+class MapRoomTestCase(TestCase):
+    def setUp(self):
+        Location.objects.create(id='1',state='서울특별시',city='강남구',dong='삼성동',
+                    detail='테헤란로 427',latitude='37.50637292',longitude='127.0536407',
+                    utmk_x='1945020.83739406',utmk_y='960738.939126282')
+        Location.objects.create(id='10',state='서울특별시',city='서초구',dong='서초동',
+                    detail='서초대로70길 51',latitude='37.49206332',longitude='127.0249488',
+                    utmk_x='1943445.64310341',utmk_y='958195.044413563')
+
+        RoomType.objects.create(id='1',name='ONE_ROOM')
+        RoomType.objects.create(id='2',name='MULTI_ROOM')
+        RoomType.objects.create(id='3',name='OFFICETEL')
```

Comment:
`bulk insert`를 사용하는 것이 테스트 속도를 더 높여줍니다~

```suggestion
        RoomType.objects.bulk_create([
            RoomType(id='1',name='ONE_ROOM')
            RoomType(id='2',name='MULTI_ROOM')
            RoomType(id='3',name='OFFICETEL')
        ])
```

Reply:
수정완료하였습니다.